kubernetes-client/src/main/java/io/fabric8/kubernetes/client/informers/cache/Reflector.java

```diff
@@ -131,16 +142,27 @@ class ReflectorWatcher implements Watcher<T> {
     @Override
     public void eventReceived(Action action, T resource) {
       if (action == null) {
+        log.error("Watcher {} v{} received null action, throwing Unrecognized event error", apiTypeClass, lastSyncResourceVersion);
```

Comment:
figured I'd add the info to the logs not the exception in case the exception message is getting swallowed somewhere